app/src/main/kotlin/com/intive/picover/images/viewmodel/ImagesViewModel.kt

```diff
@@ -15,6 +16,7 @@ import kotlinx.coroutines.launch
 class ImagesViewModel @Inject constructor(
 	private val repository: ImagesRepository,
 	private val scheduleUploadPhotoUseCase: ScheduleUploadPhotoUseCase,
+	private val snackbarHostState: SnackbarHostState,
```

Comment:
I have some doubts if we should inject `SnackbarHostState` into vm. I feel the `SnackbarHostState` is like ui component. On the other side, I guess implementation without `SnackbarHostState` in vm can be tricky cuz probably the shared viewmodel will be needed (or passing `SnackbarHostState`) to every screen (what show snakbar). What do you think about it?

Reply:
@bartek977 I see Your point here and mainly agree, but i also think, that maybe this could be a good connection for the side effects. At this moment, showing snackbars and navigating is handled in composable screens. Maybe thanks to this, we will be able to only navigate/show snackbars in the VM, test it well and avoid side effects itself and just do it in VM.

I was also thinking about alternative for this but i don't have a better idea at this moment

Reply:
To not go into the details too deeply, `SnackbarHostState` needs be a singleton anyway, so even if we would treat it as a ui component, there is no risk of leaks after configuration change, process death etc. 
Of course, we can add fancy facade, but let's keep it simple until nobody see a reason to do it in a different way :)

Reply:
@bartek977 @marcinWisnia Any conclusion here?